R/combine-categories.R

```diff
@@ -199,3 +199,45 @@ combResps <- function (subvars, combs) {
 
     return(combs)
 }
+
+
+#' Combine Categories in place
+#'
+#' This function allows you to combine the categories of a variable without
+#' making a copy of the variable.
+#' @param var A categorical Crunch variable
+#' @param from A character vector of categories you want to combine.
+#' @param to A character string with the destination category.
+#' @return the variable duly modified
+#' @export
+#' @seealso [combine()]
+collapseCategories <- function (var, from, to) {
+    if (!is.Categorical(var)) {
+        halt("Variable must be a categorical.")
+    }
+    if (!(length(to) == 1 && is.character(to))) {
+        halt("Destination category must be a character string of length 1.")
+    }
+    if (!(is.character(from))) {
+        halt(dQuote('from'), " must be a character vector.")
+    }
+    if (identical(from, to)){
+        return(var)
```

Comment:
Please add a comment explaining this early return.

Reply:
Also add a test for this case, using `expect_no_request` to assert that it's not trying to do work.